sound/soc/sof/amd/acp.h

```diff
@@ -17,6 +17,7 @@
 
 #define ACP_DSP_BAR	0
 
+#define ACP_TIMEOUT				10
```

Comment:
nit-pick: it's more of a ACP_HW_SEM_RETRY_COUNT. timeouts are usually in time units.